src/pages/SudokuPage.js

```diff
@@ -16,8 +18,9 @@ export default function SudokuPage() {
     fetch(url)
       .then((resp) => resp.json())
       .then((data) => {
-        setNumbers(data.board);
-        localStorage.setItem("currentSudoku", JSON.stringify(data.board));
+        setSudoku([...data.board]);
+        initialSudoku.current = cloneMatrix(data.board);
+
         setIsLoading(false);
```

Comment:
Hey Patrik,
you're not cathing an error here, if I see it right. might be nice to add it, just to be save. :)

Reply:
Oh yes, thanks for pointing out:)